sycl/test-e2e/lit.cfg.py

```diff
@@ -155,14 +155,18 @@
     config.available_features.add("gpu-intel-dg1")
 if lit_config.params.get("gpu-intel-dg2", False):
     config.available_features.add("gpu-intel-dg2")
-if lit_config.params.get("gpu-intel-pvc", False):
-    config.available_features.add("gpu-intel-pvc")
+if lit_config.params.get("gpu-intel-pvc", False) or \
+   lit_config.params.get("gpu-intel-pvc-vg", False) :
+    if lit_config.params.get("gpu-intel-pvc", False):
+        config.available_features.add("gpu-intel-pvc")
+    else:
+        config.available_features.add("gpu-intel-pvc-vg")
```

Comment:
Most of E2E tests using 'gpu-intel-pvc' now, need to use 'gpu-intel-pvc || gpu-intel-pvc-vg' to start on pvc-vg, right?

Reply:
If we want them to run on both PVC and PVC-VG, yes. I don't think there's any getting around that, right now as far as I see we only ever have one gpu-intel* true for a given device, having both gpu-intel-pvc and gpu-intel-pvc-vg be true would be a first and might cause problems.